app/assets/javascripts/tours.js

```diff
@@ -0,0 +1,28 @@
+/* global Tour */
+
++function () {
+  $(document).on('click', '[data-start-tour]', function (e) {
```

Comment:
Siempre tacaño con los nombres... =P

```suggestion
  $(document).on('click', '[data-start-tour]', function (event) {
```